packages/themes/src/g100.js

```diff
@@ -57,6 +57,7 @@ export const text02 = gray30;
 export const text03 = gray60;
 export const text04 = white;
 export const text05 = gray50;
+export const textError = red40;
```

Comment:
I think we'll need to add `red40` in the import block at the beginning of file, maybe adding it before `red50`?

```diff
import {
  // Blue
  blue30,
  blue40,
  blue50,
  blue60,
  blue80,

  // Gray
  gray10,
  gray30,
  gray50,
  gray60,
  gray70,
  gray80,
  gray90,
  gray100,

  // Support
  green40,
  green50,
  yellow,
+ red40,
  red50,
  red60,
  red80,
  purple40,

  // Constants
  white,

  // Tools
  rgba,
} from '@carbon/colors';
```